client/landing/gutenboarding/onboarding-block/edit.tsx

```diff
@@ -37,40 +44,81 @@ const OnboardingEdit: FunctionComponent< BlockEditProps< Attributes > > = () =>
 		window.scrollTo( 0, 0 );
 	}, [ pathname ] );
 
+	const canUseDesignSelection = useCallback( (): boolean => {
+		return ! ( ! siteVertical && ! siteTitle && ! wasVerticalSkipped );
```

Comment:
It would be a bit clearer if you inverted the logic and used an OR here `return siteVertical || siteTitle || wasVerticalSkipped` but that's just a nitpick!

Reply:
Good point, thanks for the tip. I've inverted it and added a `!!` to force a boolean and make TS happy.

Reply:
ahh yep I see 👍